x-pack/plugins/endpoint/common/types.ts

```diff
@@ -91,7 +107,14 @@ export interface AlertData {
       name: string;
     };
   };
+  process: {
+    pid: number;
+  };
   thread: {};
+
+  // Alert Details Pagination
+  next: string | null;
+  prev: string | null;
```

Comment:
Separating pagination info from the AlertEvent itself would be helpful imo.

Reply:
@peluja1012 did this.